assets/base.css

```diff
@@ -730,19 +730,37 @@ details > * {
 }
 
 .underlined-link,
-.customer a {
+.customer a,
+.header-inline-links a {
```

Comment:
Yeah, I'm all for a new name. I think `richtext-container` is a little confusing, since we already have separate `rte` link styles. Would it be too weird to name it after its function? So something like `curentcolor-links`?

Reply:
Ah no, I was just suggesting we keep the current solution exactly as it is, but call it `rte--inline`. So essentially the same as your second bullet here (but with a slightly different name). I'm cool with `inline-richtext` for now, so I'll update the PR to use that. 👍